server/src/main/java/org/apache/cassandra/sidecar/acl/AuthCache.java

```diff
@@ -119,13 +123,25 @@ public Map<K, V> getAll()
         return Collections.unmodifiableMap(cache.asMap());
     }
 
+    /**
+     * Invalidate a key.
+     * @param k key to invalidate
+     */
+    public void invalidate(K k)
+    {
+        if (cache != null)
+        {
+            cache.invalidate(k);
```

Comment:
Please add a log message when invalidating a key. It should be useful to confirm the invalidation for debugging purpose. Assuming the frequency should be low, the log level can be `INFO`. 
The method is not currently used.